project_generator/__main__.py

```diff
@@ -32,11 +32,6 @@ def main():
     # Parse Options
     parser = argparse.ArgumentParser()
 
-    parser.add_argument('-v', dest='verbosity', action='count', default=0,
```

Comment:
is this removal accidental (how does it relate to the -m support) ?

Reply:
Yes, that's an accident. Will fix it.